apps/benchmark/src/implementations/inline-styles/Dot.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { DotProps } from '../types';
+
+const Dot: React.FC<DotProps> = ({ size, x, y, children, color }) => (
+  <div
+    style={{
+      ...styles.root,
+      ...{
```

Comment:
```suggestion
```

No need for spread there

Reply:
Done in [4ed98dcb](https://github.com/microsoft/griffel/pull/132/commits/4ed98dcbe09cfc92ebbf8ba627bcbff2301fa426)